### PR TITLE
Create caro_speaking.md

### PR DESCRIPTION
**Pull request by CaroFG**

### CODE REVIEW

File: caro_speaking.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## Carolina's public speaking experiences
+
+ - Back to the living room hackathon (05/02/2021)
```
Comment (fharper) [inline caro_speaking.md:3]: Add a link to the event

File: caro_speaking.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+   MeiliSearch: the future of search engines
+   https://www.youtube.com/watch?v=4aiUCYEOVgo
+	   
+ - Rawkode Live (16/02/2021)
```
Comment (fharper) [inline caro_speaking.md:7]: add a link to his YouTube channel

**Review by fharper — COMMENTED** [review]
Can you also create a speaking.md page, with a link to this markdown, and add my name also and point to https://fred.dev/speaking please.

Also, add a link in the main index "Speaking" that point to the page I just told you about :D

**Review by fharper — CHANGES_REQUESTED** [review]
> Can you also create a speaking.md page, with a link to this markdown, and add my name also and point to https://fred.dev/speaking please.
> 
> Also, add a link in the main index "Speaking" that point to the page I just told you about :D

You are missing this one :)

**Comment by CaroFG** [comment]
Ok, so just to be sure. You want a speaking.md page with both our names and a link to the caro_speaking.md file and a link to your website. Is it correct? Should we put all those files in a folder?

**Comment by fharper** [comment]
Haha, I'll make the changes, will be simpler :)

**Review by fharper — APPROVED** [review]
LGTM

**Pull request merged**